src/ring/middleware/anti_forgery.clj

```diff
@@ -87,23 +63,33 @@
   :error-handler  - a handler function to call if the anti-forgery token is
                     incorrect or missing.
 
+  :strategy       - a state management strategy,
+                    ring.middleware.anti-forgery.session/session-strategy by
+                    default.
+
   Only one of :error-response, :error-handler may be specified."
   ([handler]
    (wrap-anti-forgery handler {}))
   ([handler options]
    {:pre [(not (and (:error-response options) (:error-handler options)))]}
-   (let [read-token    (:read-token options default-request-token)
-         error-handler (make-error-handler options)]
+   (let [strategy (:strategy options (session/session-strategy))
+         read-token (:read-token options default-request-token)
+         error-handler-fn (make-error-handler options)]
```

Comment:
Can you change this symbol back to `error-handler` as it was before? The `-fn` is redundant.

Reply:
Ok, I just disliked shadowing clojure.core/error-handler. But that's another PR.

Reply:
I think shadowing is fine so long as the meaning remains clear.

In this case, `clojure.core/error-handler` is such an obscure function that I doubt anyone would assume that `error-handler` referred to the var rather than a local symbol, especially in the context of Ring, where "handler" has a precise meaning.

Reply:
Oh, that's fine. I'm already taking it back. Just wanted to explain why I switched in the first place.